boards/arm/rpi_pico/rpi_pico-pinctrl.dtsi

```diff
@@ -31,4 +31,18 @@
 			input-schmitt-enable;
 		};
 	};
+
+	spi0_default: spi0_default {
+		group1 {
+			pinmux = <SPI0_RX_P16>, <SPI0_TX_P19>, <SPI0_SCK_P18>;
```

Comment:
TX and SCK should be in their own group, where input is not enabled

Reply:
Fix it.